### PR TITLE
internal/monitoring: add mongo prometheus monitoring.

### PR DESCRIPTION
**Pull request by fabricematrat**

### CODE REVIEW

File: internal/monitoring/monitoring.go [hunk 1]
```diff
@@ -24,4 +25,5 @@ var (
 func init() {
 	prometheus.MustRegister(requestDuration)
 	prometheus.MustRegister(uploadProcessingDuration)
+	prometheus.MustRegister(monitoring.NewMgoStatsCollector("payment"))
```
Comment (mhilton) [inline internal/monitoring/monitoring.go]: What does "payment" mean here and should it be spelled "charmstore"?

**Review by mhilton — APPROVED** [review]
LGTM with a question

File: internal/monitoring/monitoring.go (hunk 1 above)
Comment (fabricematrat) [reply internal/monitoring/monitoring.go]: @mhilton no no where do you see payment ?

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-cge.jujugui.org:8080//job/charmstore/2095/

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-cge.jujugui.org:8080//job/charmstore/2096/

**Review by jrwren — APPROVED** [review]

**Comment by fabricematrat** [comment]
:shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci-gce.jujugui.org:8080/job/charmstore-merge

**Pull request merged**